src/v2/guide/migration.md

```diff
@@ -341,13 +341,13 @@ On root Vue instances (i.e. instances created with `new Vue({ ... })`), you must
 </div>
 {% endraw %}
 
-## Computed properties
+## Propiedades computadas
 
-### `cache: false` <sup>deprecated</sup>
+### `cache: false` <sup>obsoleta</sup>
 
-Caching invalidation of computed properties will be removed in future major versions of Vue. Replace any uncached computed properties with methods, which will have the same result.
+La invalidación del almacenamiento en caché de las propiedades calculadas se eliminará en futuras versiones principales de Vue. Reemplace las propiedades calculadas no almacenadas en caché con métodos, que tendrán el mismo resultado.
```

Comment:
What do you think if we use `propiedades computadas` instead of `propiedades calculadas`? Because the title is using `computadas`. In this way we use a same way to name these properties.

Reply:
Completely agree! thanks for pointing that out